registry/api/v2/errors.go

```diff
@@ -15,6 +15,7 @@ var (
 		Message: "The operation is unsupported.",
 		Description: `The operation was unsupported due to a missing
 		implementation or invalid set of parameters.`,
+		HTTPStatusCode: http.StatusMethodNotAllowed,
```

Comment:
I'm okay with not specifying the allowed methods, here. What action could a client take that they don't already know of if they have the allowed method list?